src/Commands/ExportPostmanCommand.php

```diff
@@ -15,39 +16,63 @@ class ExportPostmanCommand extends Command
                             {--basic= : The basic auth to use on your endpoints}';
 
     /** @var string */
-    protected $description = 'Automatically generate a Postman collection for your API routes';
+    protected $description = "Automatically generate a Postman collection for your API routes";
+
+    protected ?AuthenticationMethod $authenticationMethod = null;
 
     public function handle(Exporter $exporter): void
     {
         $filename = str_replace(
-            ['{timestamp}', '{app}'],
-            [date('Y_m_d_His'), Str::snake(config('app.name'))],
-            config('api-postman.filename')
+            ["{timestamp}", "{app}"],
+            [date("Y_m_d_His"), Str::snake(config("app.name"))],
+            config("api-postman.filename")
+        );
+
+        $collectionName = str_replace(
+            "{app}",
+            config("app.name"),
+            config("api-postman.collection_name")
         );
 
-        config()->set('api-postman.authentication', [
-            'method' => $this->option('bearer') ? 'bearer' : ($this->option('basic') ? 'basic' : null),
-            'token' => $this->option('bearer') ?? $this->option('basic') ?? null,
+        $this->resolveAuth();
```

Comment:
I'm not a big fan of breaking this out. I would be in favor of doing something like this.

```
$exporter
    ->to($filename)
    ->collectionName($collectionName)
    ->setAuthentication($this->resolveAuth())
    ->export();
```

What gets returned from `resolveAuth` is the instance of the class.